captain/utils/test_sequencer/run_test_sequence.py

```diff
@@ -104,6 +149,7 @@ async def _stream_result_to_frontend(
     is_saved_to_cloud: bool = False,
     error: str | None = None,
 ):
+    logger.error(f"streaming result to frontend {state} {test_id} {result} {time_taken} {is_saved_to_cloud} {error}")
```

Comment:
why is this .error() ?

Reply:
Oh, it was to debug something. I'll remove it